src/supermarket/spec/helpers/cookbook_versions_helper_spec.rb

```diff
@@ -24,4 +24,48 @@
       expect(helper.safe_updated_at(nil)).to be <= Time.zone.now
     end
   end
+
+  describe 'chef_versions_string' do
+    let(:chef_versions) { [['12.4.1', '12.4.2'], ['11.2.3', '12.4.3']] }
```

Comment:
Looking again, I'm not sure my bit ought to be added as much as operators should be included in all version constraints appearing in the tests.